lib/io/src/ui/about_dialog.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:paintroid/io/src/ui/delete_project_dialog.dart';
+
+import '../../../ui/color_schemes.dart';
+import '../../../ui/util.dart';
```

Comment:
I am pretty sure it should be able to not be imported being dependent on the path or?

Reply:
ah.. do you mean like these imports can be removed or?

Reply:
Not removed but changed. The project is currently only using package imports and we should stay consistent :D 
That's why i would say we change those to package imports as well. I just saw that there is one more of those imports in the file landing_page.dart

Reply:
ah.. alright. Will just fix it.